### DISCUSSION TITLE
What popular dip has a name coming from the Nahuatl words "ahuacatl" and "molli?"

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#673] [Galaxy Trivia] [answered]
**Bonus:** What do these words mean?

**Comment by peterbaccaro** [comment, marked as answer]
**Guacamole** is the popular dip that comes from the Nahuatl words ahuacatl, meaning "**avocado**", and molli, meaning "**sauce**".
